core/src/test/java/kafka/testkit/KafkaClusterTestKit.java

```diff
@@ -63,7 +63,7 @@
 public class KafkaClusterTestKit implements AutoCloseable {
     /**
      * This class manages a future which is completed with the proper value for
-     * controller.connect once the randomly assigned ports for all the controllers are
+     * controller.quorum.voters once the randomly assigned ports for all the controllers are
      * known.
      */
     private static class ControllerConnectFutureManager implements AutoCloseable {
```

Comment:
can you change this class name as well to ControllerQuorumVotersFutureManager, etc.?